tc-tests-utils.sh

```diff
@@ -482,8 +482,11 @@ do_deepspeech_python_build()
     SETUP_FLAGS="--project_name deepspeech-gpu"
   fi
 
-  for pyver in ${SUPPORTED_PYTHON_VERSIONS}; do
-    pyenv install ${pyver}
+  for pyver_conf in ${SUPPORTED_PYTHON_VERSIONS}; do
+    pyver=$(echo "${pyver_conf}" | cut -d':' -f1)
+    pyconf=$(echo "${pyver_conf}" | cut -d':' -f2)
+
+    PYTHON_CONFIGURE_OPTS="--enable-unicode=${pyconf}" pyenv install ${pyver}
     pyenv virtualenv ${pyver} deepspeech
     source ${PYENV_ROOT}/versions/${pyver}/envs/deepspeech/bin/activate
```

Comment:
Not related to this PR, but FWIW, this can be just `pyenv acticate ${pyver}/envs/deepspeech`.

Reply:
activate, not acticate :)

Reply:
Yes, it should, but it always complained about something not being setup properly. I fighted that for hours, leading to nowhere. Poor error handling, poor debug handling, this was working this way so I kept it like that :)